### DISCUSSION TITLE
Add `target="_blank"` to links on the Website and in the GitHub readme

### DISCUSSION THREAD

**Discussion by CaptainException** [reflex-dev/reflex#365] [Ideas]
In the most cases when someone clicks on a link to a resource (like the docs, component library, node.js installation page, discord, twitter, etc.) they still want the original site to stay open. I checked many different links here on the github page and on the homepage and I didn't find any which opened in a new window. I assume `target="_blank"` is not used on any link. If this was changed it would make navigation easier and more convenient.

P.S.: I just noticed, that the link to python in the header of the readme links to the [image](https://camo.githubusercontent.com/9c27d165d18fc7e57ce1cd3fb94ec51d00b1c11e06dcc68e076667809e1e0564/68747470733a2f2f696d672e736869656c64732e696f2f707970692f707976657273696f6e732f70796e65636f6e652d696f2e737667) instead of the python download page.

**Comment by picklelo** [comment]
Thanks for the suggestion, we will add this!
